configs/config-tls-generic.h

```diff
@@ -297,6 +297,12 @@
 #define MBEDTLS_CIPHER_C
 #endif
 
+#if defined(CONFIG_MBEDTLS_PKCS1_V21_ENABLED)
+#define MBEDTLS_PKCS1_V21
+#endif
+
+/* Automatic dependencies */
```

Comment:
nit: Duplicated comment.

Reply:
whoops, that came in with my merge from master, sorry about that, fixed